fastscanner/benchmarks/benchmark_polygon_realtime.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+import logging
+import time
+import traceback
+
+from fastscanner.adapters.realtime.polygon_realtime import PolygonRealtime
+from fastscanner.adapters.realtime.redis_channel import RedisChannel
+from fastscanner.pkg import config
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class BenchmarkStats:
+    def __init__(self):
+        self.total_records = 0
+        self.total_flushes = 0
```

Comment:
Small nitpick. You don't need this variable as it is equal to the length of self.latencies.